src/VPolygon.jl

```diff
@@ -254,11 +254,20 @@ based on an algorithm described in [1].
 """
 function σ(d::AbstractVector{N}, P::VPolygon{N}) where {N<:Real}
     @assert !isempty(P.vertices) "the polygon has no vertices"
-    binary_or_brute_force = 10;
+    return P.vertices[_σ_helper(d, P)]
+end
+
+# heuristic to defines the method used to compute the support vector of a polygon
+# in vertex representation; if the number of vertices of the polygon is smallar
+# than this value, the brute force method is used; otherwise binary search is used
+const binary_or_brute_force = 10
+
+# return the index of a support vector of P along d from the list of vertices of P
+function _σ_helper(d::AbstractVector{N}, P::VPolygon{N}) where {N <: Real}
     if length(P.vertices) > binary_or_brute_force
```

Comment:
I wonder if we should change this:
```suggestion
     if length(P.vertices) >= binary_or_brute_force
```

Reply:
The benchmarks showed that the transition happened between 9 and 11 iirc.

cc: @SebastianGuadalupe

Reply:
This is not about the number. You can change the constant as well. It is more like what I would expect to happen if I see this constant.